diesel_cli/tests/print_schema.rs

```diff
@@ -270,6 +270,12 @@ fn print_schema_reserved_names() {
     test_print_schema("print_schema_reserved_name_mitigation_issue_3404", vec![])
 }
 
+#[test]
+#[cfg(feature = "postgres")]
+fn print_schema_regression_test_for_3446() {
```

Comment:
Is it common to name a function by his issue? Shouldn't be better to name it something like `test_print_schema_with_empty_vec()` or something like this ?

Reply:
It's a good suggestion to include also what's tested in the name. Will change that :+1: